docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,38 @@
+# Release 403 (9 Nov 2022)
+
+## General
+
+* Round `TimeWithTimeZone` values when `PARAMETRIC_DATETIME` is not set. ({issue}`14950`)
+* Fix query deadlock in multi-join queries where broadcast join size is
+  underestimated. ({issue}`14948`)
+* Fix incorrect results when `min(x, n)` or `max(x, n)` is used as a window
+  function. ({issue}`14886`)
+* Fix failure for certain queries involving joins over partitioned tables. ({issue}`14317`)
+* Fix incorrect order of parameters in `DESCRIBE INPUT` when they appear in a
+  `WITH` clause. ({issue}`14738 `)
+
+## BigQuery connector
+
+* Improve performance of `SHOW SCHEMAS` by adding a dataset metadata cache. This
+  can be configured with the `bigquery.dataset-cache-ttl` property, which has a
+  default value of `0ms` (disabled). ({issue}`14729`)
+
+## ClickHouse connector
+
+* Add support for ClickHouse `DateTime(timezone)` type in read-only operations. ({issue}`13541`)
+## Hive connector
+
+* Don't show the unsupported `AUTHORIZATION ROLE` property in the result of
+  `SHOW CREATE SCHEMA` when the access control doesn't support roles. ({issue}`8817`)
+
+## Iceberg connector
+
+* Improve performance and reduce storage costs of the `expire_snapshots` table
```

Comment:
storage cost? do you mean a dollar value? Or storage size or allocation or whatever?